highlightjs-element.html

```diff
@@ -21,63 +24,62 @@
 
     <highlightjs-element>
       <template>
-    
+
         <link rel="import" href="/components/polymer/polymer.html">
-        <polymer-element name="proto-element">
-        <template>
-          <span>I'm <b>proto-element</b>. Check out my prototype.</span>
-        </template>
-        <script>
-          Polymer({
-          });
-        </script>
-        </polymer-element>
-    
+        <dom-module name="my-element">
+          <template>
+            <span>I'm <b>my-element</b>. Check out my prototype.</span>
+          </template>
+          <script>
+            Polymer({is: 'my-element'});
+          </script>
+        </dom-module>
+
       </template>
     </highlightjs-element>
 
-@class highlightjs-element
-@blurb Element wrapper for the highlightjs library.
-@status alpha
-@snap snap.png
+@element highlightjs-element
+@group Molecules
+@hero hero.svg
+@demo demo/index.html
 -->
-<polymer-element name="highlightjs-element" attributes="text">
 
+<dom-module id="highlightjs-element">
   <template>
-
-    <style>
-      :host {
-        display: block;
-      }
-    </style>
-
-    <link href="../highlightjs/styles/default.css" rel="stylesheet" />
-    <link href="../highlightjs/styles/atelier-forest.light.css" rel="stylesheet" />
-
-    <pre id="content"></pre>
-
+    <style include="sunburst-theme"></style>
```

Comment:
What if `highlightjs-element` didn't include any styling and the host page did it instead? I'm thinking about custom styling.